monitoring/utils/metrics-exporters-with-service-monitors/litmus-metrics/chaos-exporter/chaos-exporter.yaml

```diff
@@ -17,10 +17,9 @@ spec:
         app: chaos-monitor
     spec:
       containers:
-      - image: litmuschaos/chaos-exporter:ci
-        imagePullPolicy: Always
-        name: chaos-exporter
-      serviceAccount: litmus
+        - image: litmuschaos/chaos-exporter:v1.13.x
+          imagePullPolicy: Always
+          name: chaos-exporter
       serviceAccountName: litmus
```

Comment:
I see that with portal based installations this is litmus-cluster-scope. Do we need to change this or add a comment here?

Reply:
we can add a comment for the scope but on the litmus-portal installation we would have for both scopes, this one is just for native monitoring setup